backend/src/prompts/templates/answer-user-question.j2

```diff
@@ -40,15 +41,14 @@ Reply only in JSON format with the following structure:
 ```json
 {
     "answer": "Tesla's revenue since its creation is: 2008: $15 million, 2009: $30 million, ..., 2023: $81 billion.",
-    "is_valid": true,
-    "validation_reason": "The answer includes Tesla's revenue for every year since its creation, based on the data provided."
+    "perform_web_search": false,
```

Comment:
examples do not match the explanation text
should be "should_perform_web_search"